src/linter.rs

```diff
@@ -84,10 +82,15 @@ pub fn lint_path(
     let contents = fs::read_file(path)?;
 
     // Tokenize once.
-    let tokens: Vec<LexResult> = lexer::make_tokenizer(&contents).collect();
+    let tokens: Vec<LexResult> = lexer::make_tokenizer(&contents)
+        .take_while(Result::is_ok)
+        .collect();
```

Comment:
This discards any `Err` completely, which doesn’t seem like it should be right?

Reply:
We still get errors from the parser (e.g., `foo.py:1:7: E999 SyntaxError: Got unexpected EOF` when parsing `print(`), though you're probably right. Lemme iterate on it for a sec.